acquia_lift_profiles/acquia_lift_profiles.install

```diff
@@ -180,3 +180,42 @@ function acquia_lift_profiles_update_7002(&$sandbox) {
     return t('All actions were synchronized to Lift Web');
   }
 }
+
+/**
+ * Convert existing vocabulary mappings to use taxonomy context module.
+ */
+function acquia_lift_profiles_update_7003() {
+  // 1) Convert existing mappings (if not already).
+  //
+  // Example
+  // * Before migration:
+  //   array (
+  //    'content_section' => '2',
+  //    'content_keywords' => '',
+  //    'persona' => 'taxonomy_context__country',
```

Comment:
Haha no it can't be. Unlike the examples 'content_section' and 'content_keywords', the example 'persona' here is definitely not an anticipated value. I guess I should note clear that as a part of defensive practice, this is there because the hook is expected to work even when it is somehow applied multiple times at clients' sites.